CondFormats/BTauObjects/src/BTagEntry.cc

```diff
@@ -17,8 +22,8 @@ BTagEntry::Parameters::Parameters(
   float discr_max
 ):
   operatingPoint(op),
-  measurementType(measurement_type),
-  sysType(sys_type),
+  measurementType(makeLowerCase(measurement_type)),
+  sysType(makeLowerCase(sys_type)),
```

Comment:
@davidlange6 
This utility function is only needed for the implementation of the c'tor, so it is better that the linker cannot export the function (i.e. make it visible to other translation units). Maybe anonymous namespaces are better in C++...
Anyhow, your suggestion looks good: we can do the following:

1. in the initialiser list, initialise `measurementType` with `measurement_type` and sysType with sys_type
1. in the c'tor body, transform them as lowercase.

````
BTagEntry::Parameters::Parameters(
  OperatingPoint op,
  std::string measurement_type,
  std::string sys_type,
  JetFlavor jf,
  float eta_min,
  float eta_max,
  float pt_min,
  float pt_max,
  float discr_min,
  float discr_max
):
  operatingPoint(op),
  measurementType(measurement_type),
  sysType(sys_type),
  jetFlavor(jf),
  etaMin(eta_min),
  etaMax(eta_max),
  ptMin(pt_min),
  ptMax(pt_max),
  discrMin(discr_min),
  discrMax(discr_max) {
  std::transform(measurementType.begin,measurementType.end(),measurementType.begin(), ::tolower);
  std::transform(sysType.begin,sysType.end(),sysType.begin(), ::tolower);
}
````
What do you think?